### PR TITLE
Salt Bundle: wording improvements

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Cosmetic wording improvements.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 1]
```diff
@@ -3,11 +3,12 @@
 
 == What is Salt Bundle?
 
-The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, additional Python modules, and libraries.
```
Comment (vzhestkov) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:6]: Not sure if it's a proper rewording as there is only the modules required for salt, what are the modules additional to? We just imlementing the ways to add additional modules inside with `pip`.

**Review by vzhestkov — CHANGES_REQUESTED** [review]
@keichwa thanks for improving it.
Just one point I don't like in the comments. I would suggest not to touch this part yet as there are really no any additionals but just only required modules for Salt.

**Review by jcayouette — CHANGES_REQUESTED** [review]
Please fix the vale style check item. :) Thanks!

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-saltbundle.adoc]: ok, I reverted this

**Review by vzhestkov — APPROVED** [review]
Thanks :+1:

**Comment by keichwa** [comment]
> Please fix the vale style check item. :) Thanks!

this requires an own PR: https://github.com/uyuni-project/uyuni-docs/pull/1487

**Comment by keichwa** [comment]
> > Please fix the vale style check item. :) Thanks!
> 
> this requires an own PR: #1487

I merged that one.

**Comment by keichwa** [comment]
@vzhestkov I forgot to ask... Do we want this in 4.2, too?

**Comment by vzhestkov** [comment]
If I remember right the plan is to add the support for of Salt Bundle to 4.2 after releasing 4.3 to make it possible to use the same client tools.
@admd am I right?

**Comment by admd** [comment]
Yes, that is correct Victor @vzhestkov

**Pull request merged**